src/main/java/UserSearchUI.java

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
It's good that you started a Javadoc for a class -- as you may know, you can also add Javadoc for methods (and variables).

Reply:
Will add all the documentation soon! I'm just focusing on making the code work first!